plugins/catalog-backend-module-terraform-cloud/package.json

```diff
@@ -0,0 +1,43 @@
+{
+  "name": "@backstage/plugin-catalog-backend-module-terraform-cloud",
+  "version": "0.1.0",
+  "main": "src/index.ts",
+  "types": "src/index.ts",
+  "license": "Apache-2.0",
+  "private": true,
+  "publishConfig": {
+    "access": "public",
+    "main": "dist/index.cjs.js",
+    "types": "dist/index.d.ts"
+  },
+  "scripts": {
+    "start": "backstage-cli backend:dev",
+    "build": "backstage-cli backend:build",
+    "lint": "backstage-cli lint",
+    "test": "backstage-cli test",
+    "prepack": "backstage-cli prepack",
+    "postpack": "backstage-cli postpack",
+    "clean": "backstage-cli clean"
+  },
+  "dependencies": {
+    "@backstage/backend-common": "^0.9.11",
+    "@backstage/catalog-model": "^0.9.8",
+    "@backstage/config": "^0.1.8",
+    "@backstage/plugin-catalog-backend": "^0.19.2",
+    "@types/express": "*",
+    "cross-fetch": "^3.0.6",
+    "express": "^4.17.1",
+    "express-promise-router": "^4.1.0",
```

Comment:
I think it's worthwhile cleaning up a few of these dependencies that aren't used